1_2.js

```diff
@@ -0,0 +1,10 @@
+while (true) {
+  var city = prompt("日本の首都は？");
+
+  if (city === null)
+    break;
```

Comment:
そもそもちゃんと入力されているか？のチェックを先に済ませて、
問題がある場合はその時点でエラーを表示してしまった方が
流れを追いやすいコードになるかと思います。

```suggestion
  if (city === null) {
    alert("不正解です！");
    break;
  }
```